config/options.go

```diff
@@ -564,3 +633,30 @@ func HandleConfigUpdate(configFile string, opt *Options, services []OptionsUpdat
 	}
 	return newOpt
 }
+
+// homeDir returns the best guess of the current user's home
+// directory from environment variables. If unknown, "." (the
+// current directory) is returned instead.
+func homeDir() string {
```

Comment:
you can use https://github.com/golang/go/blob/go1.14.2/src/os/file.go#L469

Reply:
Oh very neat!